core/src/main/java/com/sequenceiq/cloudbreak/service/stack/CreateCloudResourceService.java

```diff
@@ -0,0 +1,55 @@
+package com.sequenceiq.cloudbreak.service.stack;
+
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cloud.event.model.EventStatus;
+import com.sequenceiq.cloudbreak.cloud.event.platform.CreateCloudNetworkRequest;
+import com.sequenceiq.cloudbreak.cloud.event.platform.CreateCloudNetworkResult;
+import com.sequenceiq.cloudbreak.cloud.model.CreatedCloudNetwork;
+import com.sequenceiq.cloudbreak.domain.Credential;
+import com.sequenceiq.cloudbreak.service.OperationException;
+import com.sequenceiq.flow.reactor.ErrorHandlerAwareReactorEventFactory;
+
+import reactor.bus.EventBus;
+
+@Service
+public class CreateCloudResourceService {
```

Comment:
`CloudResourceCreationService` would be grammatically a better name. But why CloudResource? Why not CloudNetwork? `CloudNetworkCreationService`